src/jtagmkII.c

```diff
@@ -620,27 +620,9 @@ int jtagmkII_recv(const PROGRAMMER *pgm, unsigned char **msg) {
        */
       memmove(*msg, *msg + 8, rv);
 
-      if (verbose == 4)
-      {
-          int i = rv;
-          unsigned char *p = *msg;
-          pmsg_trace("recv: ");
-
-          while (i) {
-            unsigned char c = *p;
-            if (isprint(c)) {
-              msg_trace("%c ", c);
-            }
-            else {
-              msg_trace(". ");
-            }
-            msg_trace("[%02x] ", c);
-
-            p++;
-            i--;
-          }
-          msg_trace("\n");
-      }
+      if(verbose > 3)
+        trace_buffer("jtagmkII_recv: ", *msg, rv);
```

Comment:
Perhaps use `__func__` here?

Reply:
No, that's not as straight-forward as I first thought. Nevermind...

Reply:
That's actually a good idea - done